backends-common/cassandra/src/main/java/org/apache/james/backends/cassandra/versions/CassandraSchemaVersionManager.java

```diff
@@ -19,6 +19,11 @@
 
 package org.apache.james.backends.cassandra.versions;
 
+import static org.apache.james.backends.cassandra.versions.CassandraSchemaVersionManager.SchemaState.UP_TO_DATE;
```

Comment:
done I mean I reorginize import using idea. I don't think it remove any import but if there were one it did it